course/permissions.py

```diff
@@ -248,7 +248,8 @@ def has_object_permission(self, request: HttpRequest, view: Any, module: CourseM
         if module.status == CourseModule.STATUS.HIDDEN:
             return False
 
-        if not module.is_after_open():
+        #if this isn't an enrollment question and the module hasn't opened yet
+        if not (hasattr(view, 'exercise') and view.exercise.status in ('enrollment', 'enrollment_ext')) and not module.is_after_open():
```

Comment:
Should we rather use
```
view.exercise.status in (LearningObject.STATUS.ENROLLMENT, LearningObject.STATUS.ENROLLMENT_EXTERNAL)
```
Perhaps could also be split into multiple lines, as this line is getting rather long